tables-test-fixtures/tables-test-fixtures-iceberg-1.2/src/main/java/com/linkedin/openhouse/tablestest/TestSparkSessionUtil.java

```diff
@@ -18,44 +18,47 @@ private TestSparkSessionUtil() {
 
   public static SparkSession create(String customCatalogName, URI tablesServiceURI, URI fsURI)
```

Comment:
nit: proposed approach runs well, another way we could've accepted config is with following:
```
 public static SparkSession create(String customCatalogName, URI tablesServiceURI, URI fsURI, Map<String, String> customConfs)
```
which i find simpler to understand, will let you make the call.